pkg/ccl/crosscluster/physical/stream_ingestion_dist_test.go

```diff
@@ -145,6 +126,12 @@ func TestMeasurePlanChange(t *testing.T) {
 			after:  makePlan(makeProc(1, []int{2}), makeProc(2, []int{1})),
 			frac:   0,
 		},
+		{
+			name:   "lots of processors",
```

Comment:
does this fail when your patch is commented out?

Reply:
Yup